infrastructure/migrations/0042_project_project_capacity_timeframe.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-02-21 16:52
+from __future__ import unicode_literals
+import sys
+
+from django.db import migrations, models
+from infrastructure.models import Project, ProjectTimeFrameUnits
+
+
+def update_powerplant_project_capacity_timeframe_unit(apps, schema_editor):
+    if 'test' in sys.argv:
+        return
+    projects = Project.objects.filter(
```

Comment:
This should use ``apps.get_model`` to get the [correct version of the model in the migration timeline](https://docs.djangoproject.com/en/2.1/topics/migrations/#data-migrations). If these migrations are run in the future and the current ``Project`` has changed drastically, the direct import may fail here.